config/clusters/nasa-ghg/common.values.yaml

```diff
@@ -19,9 +19,9 @@ basehub:
       homepage:
         templateVars:
           org:
-            name: "NASA Green House Gases Center"
+            name: "U.S. Greenhouse Gas Center"
             logo_url: https://raw.githubusercontent.com/US-GHG-Center/ghgc-docs/87204d4a4e5f29c335a8f905c73de551aabd4845/Logo/ghg-logo.svg
```

Comment:
this is still the old logo, do we wanna replace it with the placeholder?

Reply:
Yes, please.